docs/install/install-from-package/install-manager.rst

```diff
@@ -51,6 +62,7 @@ would be:
    ssl-enabled = false
 
    heartbeat-timeout = 30.0
```

Comment:
Let's update `heartbeat-timeout` to `40.0` since the heart beat period from agent is increased in 23.09.

Reply:
fixed it.